fs2/gst/attn.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2019 Shigeki Karita
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
Please update the LICENSE files both in this repo and in EV to list this new exception.